lib/declare_schema.rb

```diff
@@ -81,12 +81,22 @@ def normalize_collation(collation)
 
     def default_charset=(charset)
       charset.is_a?(String) or raise ArgumentError, "charset must be a string (got #{charset.inspect})"
-      @default_charset = normalize_charset(charset)
+      @default_charset_before_normalization = charset
+      @default_charset = nil
+    end
+
+    def default_charset
+      @default_charset || normalize_charset(@default_charset_before_normalization)
```

Comment:
❓ Should this be an `||=`?

Reply:
Yes, good catch! Both fixed here: https://github.com/Invoca/declare_schema/pull/166/commits/2def468ac7d426a0d4c29df2b2a50063eabf36f9